samples/python/video.py

```diff
@@ -115,35 +121,37 @@ def __init__(self, **kw):
         white_quads = []
         black_quads = []
         for i, j in np.ndindex(sy, sx):
-            q = [[j, i, 0], [j+1, i, 0], [j+1, i+1, 0], [j, i+1, 0]]
+            q = [[j, i, 0], [j + 1, i, 0], [j + 1, i + 1, 0], [j, i + 1, 0]]
             [white_quads, black_quads][(i + j) % 2].append(q)
         self.white_quads = np.float32(white_quads)
         self.black_quads = np.float32(black_quads)
 
         fx = 0.9
-        self.K = np.float64([[fx*w, 0, 0.5*(w-1)],
-                        [0, fx*w, 0.5*(h-1)],
-                        [0.0,0.0,      1.0]])
+        self.K = np.float64(
+            [[fx * w, 0, 0.5 * (w - 1)], [0, fx * w, 0.5 * (h - 1)], [0.0, 0.0, 1.0]]
```

Comment:
Just a wrong automatic change. Code should be readable by humans and they should see here 3x3 matrix.